docs/examples/SignalCore.ipynb

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
there seems to be an issue with the notebook as well that it's not a valid JSON, see here https://github.com/QCoDeS/Qcodes_contrib_drivers/blob/d8b66cd63692e22267ec29a2910516f8ce5cb18d/docs/examples/SignalCore.ipynb

Reply:
Yep that's me adding:
```json

            "nbsphinx":{
               "execute":"never"
            }
```
too fast...
Thanks for the catch!